resources/docs/simulation/creating-simulations/h.cloud.md

```diff
@@ -19,6 +19,7 @@ Single-run simulations are not supported on hCloud at this time. Please ensure y
 
 Simulations within **hCore** can be run in one of two ways: locally inside your browser, or at scale on our **hCloud** compute platform.
 
+<!-- prettier-ignore -->
```

Comment:
Prettier aligns table markup in source and there is no option to disable it. I’ve added `<!-- prettier-ignore -->` for now to reduce the diff. We can review this later.